Provider/FileProvider.php

```diff
@@ -334,6 +334,14 @@ public function validate(ErrorElement $errorElement, MediaInterface $media)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function configureMultiUpload(FormMapper $formMapper)
+    {
+        $formMapper->add('binaryContent', 'file', array('attr' => array('multiple' => true)));
```

Comment:
please do it like here:
https://github.com/sonata-project/SonataMediaBundle/pull/1105/files#r104668683